application/cms/models.py

```diff
@@ -313,6 +313,25 @@ def primary_data_source(self):
     def secondary_data_source(self):
         return self.data_sources[1] if len(self.data_sources) >= 2 else None
 
+    # A short summary of the page exposed as metadata for use by search
```

Comment:
Ideally, comments for methods/properties would be declared on the first line inside the method within triple-quoted strings. This allows Python to automatically convert them to docstrings, and is more conventional. We aren't very consistent about this, but if you could move it inside that would be good.

Reply:
Dang, I knew this but forgot it. Moved.